app/views/suggestions/_suggestion.html.erb

```diff
@@ -0,0 +1,12 @@
+<h1> <%=@heading %> </h1>
+
+<% Food.hashify(@data).each do |suggestion| %>
```

Comment:
This logic belongs in the controller. You should have all your variables set there, and then just access that data once you're in the view. Set `food` and `music` in your `Suggestions Controller` and then just start on line 6 here.